universal-login-relayer/lib/integration/ethereum/services/AuthorisationService.ts

```diff
@@ -0,0 +1,22 @@
+import {recoverFromCancelAuthorisationRequest, CancelAuthorisationRequest, hashCancelAuthorisationRequest} from '@universal-login/commons';
+import { ethers, providers} from 'ethers';
+import WalletMasterWithRefund from '@universal-login/contracts/build/WalletMasterWithRefund.json';
+import { UnauthorisedAddress } from '../../../core/utils/errors';
+
+class AuthorisationService {
+  constructor(private provider: providers.Provider) {}
+
+  async isValidSignature(cancelAuthorisationRequest: CancelAuthorisationRequest) {
```

Comment:
should be ensureValidSignature (throws exception if invalid)
By convention isValidSignature return false if invalid